src/actions/index.js

```diff
@@ -66,12 +66,14 @@ export const getPlayerTotals = (accountId, params) => action('playerTotals', pro
 export const getPlayerMmr = (accountId, params) => action('playerMmr', process.env.REACT_APP_API_HOST, `api/players/${accountId}/ratings`, params);
 export const getPlayerRankings = (accountId, params) => action('playerRankings', process.env.REACT_APP_API_HOST, `api/players/${accountId}/rankings`, params, transformRankings);
 export const getStrings = () => async (dispatch) => {
+  const getLang = (lang) => langs.find(item => item.value === lang);
   const savedLang = window.localStorage && window.localStorage.getItem('localization');
+  const userLang = window.navigator.language;
   const defaultLang = langs[0];
-  const selectedLang = langs.find(lang => lang.value === savedLang) || langs[0];
-  const defData = await import(`../lang/${defaultLang.value}.json`);
-  const selData = await import(`../lang/${selectedLang.value}.json`);
-  dispatch({ type: 'strings', payload: { ...defData, ...selData } });
+  const lang = getLang(savedLang) || getLang(userLang) || defaultLang;
+  const data = await import(`../lang/${lang.value}.json`);
+
+  dispatch({ type: 'strings', payload: { ...data } });
```

Comment:
isn't this still going to import one set of strings? the problem is that we need to merge the translated strings with the english strings as the files don't get the new keys immediately

Reply:
hm really, i forgot about fallback :)